src/cleo/io/outputs/section_output.py

```diff
@@ -84,6 +84,7 @@ def _write(self, message: str, new_line: bool = False) -> None:
 
         super()._write(message, new_line=True)
         super()._write(erased_content, new_line=False)
+        return None
```

Comment:
👎 I think the code above this should instead do
```py
if not self.is_decorated():
    super()._write(message, new_line=new_line)
    return
```
to not trigger the rule

Reply:
is it better now?

Reply:
Yup, definitely 😄